pkg/tenant/db/members.go

```diff
@@ -22,29 +22,35 @@ const (
 )
 
 type Member struct {
-	OrgID        ulid.ULID    `msgpack:"org_id"`
-	ID           ulid.ULID    `msgpack:"id"`
-	Email        string       `msgpack:"email"`
-	Name         string       `msgpack:"name"`
-	Role         string       `msgpack:"role"`
-	Status       MemberStatus `msgpack:"status"`
-	Created      time.Time    `msgpack:"created"`
-	Modified     time.Time    `msgpack:"modified"`
-	DateAdded    time.Time    `msgpack:"date_added"`
```

Comment:
Changing `DateAdded` to `JoinedAt` will require a migration to transform all the msgpack members already in the database. I'll leave it to you to either change this field back or to create a story for the migration.